inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/project/ProjectKnowledgeBasePanel.properties

```diff
@@ -41,6 +40,9 @@ kb.wizard.steps.type.name=Name
 kb.wizard.steps.type.type=Type
 kb.wizard.steps.schema=Schema
 kb.wizard.steps.local.fileupload=File(s)
+kb.wizard.steps.local.predefinedKBs = Pre-defined knowledge bases
+kb.wizard.steps.local.schemaOnMouseOver = Schema: {0}\nAccess: {1}
```

Comment:
Does `\n` actually have an effect here? Browsers usually ignore line breaks in HTML. If this shows in the browser, it might have to be a  `<br>` instead.

Reply:
Yes, the \n works.